### PR TITLE
Fix visual glitches in Calypso

### LINKED ISSUE #56: Visual glitch in stats
It seems like some CSS leak is causing broken layout in Stats visitors: 

warnings

This is displayed fine in Calypso master.

### PR DESCRIPTION
**Pull request by Copons**
Fixes #56 

Namespace the form extend classes to prevent CSS leaking to Calypso.

### CODE REVIEW

File: src/boot/stylesheets/shared/forms.scss [hunk 1]
```diff
@@ -20,72 +9,40 @@
...
-fieldset,
-input[type="text"],
-input[type="email"],
-input[type="number"],
-input[type="password"],
-textarea,
-select,
-label {
-  box-sizing: border-box;
-}
```
Comment (vindl) [inline src/boot/stylesheets/shared/forms.scss]: Missing newline.

File: src/boot/stylesheets/shared/forms.scss [hunk 2]
```diff
@@ -20,72 +9,40 @@
...
   }
 }
 
-%textarea {
+%wpnc-textarea {
   min-height: 92px;
 }
-
-
-// ==========================================================================
```
Comment (vindl) [inline src/boot/stylesheets/shared/forms.scss]: Will the removal of this block affect `iFrame` notes versions?

Comment (Copons) [inline src/boot/stylesheets/shared/forms.scss]: Tested it and it works, but I'd be much more comfortable with a second look on that!

Comment (vindl) [inline src/boot/stylesheets/shared/forms.scss]: Maybe if we just keep everything in this file as is, but nest it in `wpnc_main`?

**Pull request merged**